kubernetes/create-services.sh.j2

```diff
@@ -34,3 +33,4 @@ kubectl create --record --filename services/nginx-proxy.yml
 ./wait-until-ready.sh conductor-worker     30
 ./wait-until-ready.sh nginx-storage        30
 ./wait-until-ready.sh nginx-proxy          30
+./wait-until-ready.sh geth                 60
```

Comment:
If this is going to wait for geth to sync then it might not be the best idea. Since it often takes more than 60 seconds, this will cause a lot of failed deployments.

Reply:
What do you suggest in this situation ?

Reply:
The problem is when deploying process is end, when the all pods are on the cluster or when the all pods are ready to communicate with each other without error.

On which level we want to catch the problem with geth. Suspend deployment to wait for geth is ready to go or some concent service which is check connection to geth.

Reply:
OK. It's not a problem to solve now. Let's leave it as is and we'll worry about it when we run into problems with it.